deploy/manager/manager.yaml

```diff
@@ -37,6 +37,8 @@ spec:
                   fieldPath: metadata.name
             - name: OPERATOR_NAME
               value: "governance-policy-propagator"
+            - name: CONTROLLER_CONFIG_REQUEUE_ERROR_DELAY
+              value: "1"
```

Comment:
(Question/Optional) Is this necessary or just to make tests run faster? If the latter, maybe we leave it out of `deploy/` and leave it in the `Makefile`?

Reply:
Just tests, so that the one test doesn't need a 10 minute timeout. Do you know if the `deploy/` files are used for anything really, outside of our tests? Even if they are, this isn't a terrible setting, IMO.

Reply:
I don't think so? But even for our tests, I think the controller is run locally, which is why I was wondering.